src/remarx/sentence/corpus/tei_input.py

```diff
@@ -37,6 +37,25 @@ class BaseTEIXmlObject(xmlmap.XmlObject):
     ROOT_NAMESPACES: ClassVar[dict[str, str]] = {"t": TEI_NAMESPACE}
 
 
+class TEIFootnote(BaseTEIXmlObject):
+    """XmlObject wrapper for footnotes with convenience accessors."""
+
+    first_line_break = xmlmap.StringField("(.//t:lb[@n])[1]/@n")
+    "Line number of the first TEI line break (`lb`) within this footnote"
+
+    def get_first_line_number(self) -> int:
+        """Return the first line number for this footnote."""
+        return int(self.first_line_break)
```

Comment:
`xmlmap` includes an integer field; let's simplify this code. Here's a suggestion:

```suggestion
    line_number = xmlmap.IntegerField("(./t:lb[1]/@n")
    "Line number where this footnote begins, based on first TEI line beginning (`lb`) within this note"
```